src/main/java/io/keyko/nevermined/api/AgreementsAPI.java

```diff
@@ -53,4 +41,13 @@
      * @throws ServiceAgreementException Exception
      */
     boolean isAccessGranted(String agreementId, DID did, String consumerAddress) throws ServiceAgreementException;
+
+    /**
+     * Get the status of a service agreement.
+     *
+     * @param agreementId id of the agreement
+     * @return AgreementStatus with condition status of each of the agreement's conditions.
```

Comment:
This now returns the `Agreement` and not only the status

Reply:
adding more details to the documentation